test/scripts/setup_test_environment.sh

```diff
@@ -2,8 +2,11 @@
 
 set -ev
 
-chmod ugo+x ./test/scripts/install_chromedriver_travis.sh
-sudo ./test/scripts/install_chromedriver_travis.sh
+# run only on CI, pull requests and when NODE_ENV=test
+if [[ ${CI} == "true" || (${NODE_ENV} = "test" && ${TRAVIS_PULL_REQUEST} != "false") ]]; then
```

Comment:
mmm bash

Reply:
good you pointed out, just found out there was a mistake in condition :)